python/ray/worker.py

```diff
@@ -1125,22 +1126,14 @@ def _initialize_serialization(driver_id, worker=global_worker):
     serialization_context.set_pickle(pickle.dumps, pickle.loads)
     pyarrow.register_torch_serialization_handlers(serialization_context)
 
-    # Define a custom serializer and deserializer for handling Object IDs.
-    def object_id_custom_serializer(obj):
-        return obj.binary()
-
-    def object_id_custom_deserializer(serialized_obj):
-        return ObjectID(serialized_obj)
-
-    # We register this serializer on each worker instead of calling
-    # register_custom_serializer from the driver so that isinstance still
-    # works.
-    serialization_context.register_type(
-        ObjectID,
-        "ray.ObjectID",
-        pickle=False,
-        custom_serializer=object_id_custom_serializer,
-        custom_deserializer=object_id_custom_deserializer)
+    for id_type in [
+            UniqueID, ObjectID, DriverID, ActorID, ActorHandleID, FunctionID,
+            ActorClassID, ClientID, TaskID
```

Comment:
I found this list is duplicated in many places. Should we make it a constant? Otherwise, when someone add a new ID type, it's likely that they will forget to modify this.